svc-managers/rpc/manager_test.go

```diff
@@ -108,7 +108,7 @@ func mockGetPluginData(pluginID string) (mgrmodel.Plugin, *errors.Error) {
 	}, nil
 }
 
-func mockUpdateManagersData(key string, managerData map[string]interface{}) error {
+func mockUpdateData(key string, updateData map[string]interface{}, table string) error {
```

Comment:
This needs an implementation

Reply:
There is a separate test for UpdateData however in this code it is only mocking the response